internal/publish/model/exposure_model_test.go

```diff
@@ -445,6 +445,14 @@ func TestStillValidKey(t *testing.T) {
 			if !tf[0].CreatedAt.Equal(tc.createdAt) {
 				t.Errorf("wrong createdAt time, want: %v got: %v", tc.createdAt, tf[0].CreatedAt)
 			}
+
+			wantStats := &PublishInfo{
```

Comment:
I've been trying to move us away from go-cmp for simple struct comparisons, since it's not exactly clear what you're trying to test.